fero/analysis.py

```diff
@@ -548,7 +661,7 @@ def _build_optimize_request(
         return opt_request
 
     def _request_prediction(
-        self, prediction_request: dict, synchonous: bool
+        self, prediction_request: dict, synchronous: bool
```

Comment:
I'm shocked I made a spelling mistake, shocked.

Reply:
EDIT: "I'm shocked I made a spelling mistake, shoked."

Reply:
Honestly, I'm more surprised that I found it